src/scenes/Partners/Detail/PartnerDetail.tsx

```diff
@@ -60,21 +113,98 @@ export const PartnerDetail = () => {
                 <Skeleton width="75%" />
               )}
             </Typography>
-            {/* {partner ? (
+            <Tooltip title="Update Global Innovations Client">
               <IconButton
                 color="primary"
                 aria-label="edit partner"
-                onClick={editPartner}
+                onClick={() => editPartner('globalInnovationsClient')}
               >
                 <PencilCircledIcon />
               </IconButton>
```

Comment:
Use FAB. Tooltip should match aria-label.

Reply:
Changed.  I haven't used many FABs before this, what's the difference between this and a regular icon button?  From what I've read looks like they're mostly for those fixed position buttons on mobile screens?